<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-10-18T12:51:00 IST" name="Suite" finished-at="2024-10-18T12:51:35 IST" duration-ms="34589">
    <groups>
    </groups>
    <test started-at="2024-10-18T12:51:00 IST" name="Test" finished-at="2024-10-18T12:51:35 IST" duration-ms="34589">
      <class name="TestCases.VerifyLoginTestCase">
        <test-method is-config="true" signature="LaunchBrowser()[pri:0, instance:TestCases.VerifyLoginTestCase@29215f06]" started-at="2024-10-18T12:51:00 IST" name="LaunchBrowser" finished-at="2024-10-18T12:51:14 IST" duration-ms="13970" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LaunchBrowser -->
        <test-method signature="login()[pri:0, instance:TestCases.VerifyLoginTestCase@29215f06]" started-at="2024-10-18T12:51:14 IST" name="login" finished-at="2024-10-18T12:51:15 IST" duration-ms="717" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method is-config="true" signature="CloseBrowser()[pri:0, instance:TestCases.VerifyLoginTestCase@29215f06]" started-at="2024-10-18T12:51:15 IST" name="CloseBrowser" finished-at="2024-10-18T12:51:15 IST" duration-ms="770" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CloseBrowser -->
      </class> <!-- TestCases.VerifyLoginTestCase -->
      <class name="TestCases.VerifySignUpTestCase">
        <test-method is-config="true" signature="LaunchBrowser()[pri:0, instance:TestCases.VerifySignUpTestCase@368247b9]" started-at="2024-10-18T12:51:15 IST" name="LaunchBrowser" finished-at="2024-10-18T12:51:22 IST" duration-ms="6986" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LaunchBrowser -->
        <test-method signature="Signup()[pri:0, instance:TestCases.VerifySignUpTestCase@368247b9]" started-at="2024-10-18T12:51:22 IST" name="Signup" finished-at="2024-10-18T12:51:34 IST" duration-ms="11446" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [Free CRM Trial: Salesforce 30-Day Trial - Salesforce IN] but found [Start Your 30-Day Free Trial - Salesforce IN]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Free CRM Trial: Salesforce 30-Day Trial - Salesforce IN] but found [Start Your 30-Day Free Trial - Salesforce IN]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at Resources.CommonUtilities.handleAssertion(CommonUtilities.java:20)
at TestCases.VerifySignUpTestCase.Signup(VerifySignUpTestCase.java:43)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Signup -->
        <test-method is-config="true" signature="CloseBrowser()[pri:0, instance:TestCases.VerifySignUpTestCase@368247b9]" started-at="2024-10-18T12:51:34 IST" name="CloseBrowser" finished-at="2024-10-18T12:51:35 IST" duration-ms="655" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CloseBrowser -->
      </class> <!-- TestCases.VerifySignUpTestCase -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
